schema/schema.py

```diff
@@ -51,15 +52,14 @@ def format_param(param: click.core.Option) -> Dict[str, Any]:
     return formatted_param
 
 
-def get_params_from_command(cli, main_command_name: str = "") -> Dict[str, dict]:
+def get_params_from_command(cli) -> Dict[str, dict]:
     """Given a command CLI, return it in a dictionary, pointing to its parameters as dictionary objects."""
     params = [
         param
         for param in cli.params
         if param.name and isinstance(param, click.core.Option)  # exclude None and non-Options
     ]
-    cmd_name = SamConfig.to_key([main_command_name, cli.name]) if main_command_name else cli.name
-    return {cmd_name: {param.name: format_param(param) for param in params}}
+    return {param.name: format_param(param) for param in params if param.name}
```

Comment:
Could we refactor this to be a single loop instead of two? Or if should stay, why we couldn't move the parameter collection and parsing to a single loop